pages/tarieven.vue

```diff
@@ -40,46 +40,46 @@
             </li>
           </ul>
           <p>
-            Pas op: aan de tarieven kunnen geen rechten ontleend worden, voordat we aan de slag gaan maken we definitieve afspraken.
+            Pas op: aan de tarieven kunnen geen rechten ontleend worden, voordat we aan de slag gaan, maken we definitieve afspraken.
           </p>
         </details>
         <div class="page-content__spacer"></div>
         <h2 class="heading-page">
           Appje Eitje Werkwijze
         </h2>
         <h3>
-          Geen factuurstress
+          Aan de slag als alles duidelijk is
         </h3>
         <p>
-          Geen verrassingen meer.
-          We werken op nacalculatie.
-          Gemaakte uren kun je volgen per maand, via onze tools.
-          Aan het einde van de maand corrigeren we dit en factureren na akkoord.
+          Wij helpen je om het (ontwikkel)werk in zinvolle delen op te splitsen en zorgen zo samen voor een backlog.
+          Als alles duidelijk is, kunnen we actief aan de slag!
```

Comment:
Bedenk me, hoe is zoiets als dit? 
```suggestion
          Zodra alles duidelijk is van een item, gaan we aan de slag!
```

Reply:
Of dit?

```suggestion
          Wanneer er geen vragen meer zijn, gaan we aan de slag!
```